Doc/c-api/structures.rst

```diff
@@ -482,6 +519,63 @@ definition with the same method name.
    Equivalent to ``PyCMethod_New(ml, self, NULL, NULL)``.
 
 
+.. c:function:: int PyCFunction_GetFlags(PyObject *func)
+
+   Get the function's flags on *func* as they were passed to
+   :c:member:`~PyMethodDef.ml_flags`.
+
+   If *func* is not a C function object, this fails with a
+   :class:`SystemError`.
```

Comment:
This one is a regular exception right? so no need to mention it. What I wanted to know is whether passing `func == NULL` produced a SIGSEGV or a Python exception. Since passing NULL is "allowed" (but raises an exception) it's fine to remove the mention to SystemError.